# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../exml/pc_requirements.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- exml/configure.in.orig	2006-06-01 14:49:42.000000000 +0200
+++ exml/configure.in	2006-06-01 15:00:33.000000000 +0200
@@ -82,6 +82,9 @@
 AC_SUBST(xml2_cflags)
 AC_SUBST(xml2_libs)
 
+requirements="ecore libxml-2.0 $requirements"
+AC_SUBST(requirements)
+
 AC_OUTPUT([
 Makefile 
 exml.pc
